### DISCUSSION TITLE
Question about ProxyShape displaying different rendering result In Maya2019 At Different Commit

### DISCUSSION THREAD

**Discussion by whitemoonstone** [Autodesk/maya-usd#1336] [Q&A]
Hi,
Recently, I built **maya-usd** using CMake with USD19.11 and found that ProxyShape imported by **AL_USD** plugin showing different rendering result.

At commit [77425bc485a870c41a934f57d14ab76579105863](https://github.com/Autodesk/maya-usd/commit/77425bc485a870c41a934f57d14ab76579105863)
The imported USD proxyshape rendering is like this 
![aa41c555](https://user-images.githubusercontent.com/37799121/114689933-cd26a480-9d48-11eb-9ca4-b1ab12e304c3.png)
While at commit [70efd232c05f9d5f5d880136b56fe435e996be60](https://github.com/Autodesk/maya-usd/commit/70efd232c05f9d5f5d880136b56fe435e996be60)
The rendering is like 
![70efd2](https://user-images.githubusercontent.com/37799121/114690177-095a0500-9d49-11eb-9da0-7682a0de8d97.png)

May I ask which code you have modified/added at which commit to implementing these rendering difference at [77425bc48](https://github.com/Autodesk/maya-usd/commit/77425bc485a870c41a934f57d14ab76579105863)

Thanks in advance @kxl-adsk !

**Comment by kxl-adsk** [comment]
Hi @whitemoonstone,

Thank you for your question. You might have noticed that I converted the issue to a discussion thread. In the future, please use discussions when you want to connect with us and the community.

For your question, these two commits are very old...using GitHub compare tool we can see what went in - https://github.com/Autodesk/maya-usd/compare/70efd232c05f9d5f5d880136b56fe435e996be60...77425bc485a870c41a934f57d14ab76579105863

You can see that we added VP2RenderDelegate as a new option for rendering proxy shapes.

Cheers,
KxL
